docs/user_guide/readthedocs.md

```diff
@@ -2,6 +2,18 @@
 
 This theme comes with support for {{ rtd }}, a popular service for hosting documentation in the scientific Python community.
 
+## Version switcher
+
+Rather than using the [version switcher that this theme provides](version-dropdown.rst),
+this theme also supports the ReadTheDocs version switcher by default.
```

Comment:
```suggestion
Projects hosted on {{ rtd }} can use the {{ rtd }} supplied version switcher instead of the [version switcher that this theme provides](version-dropdown.rst).
Its presence will be automatically detected by this theme, and placed in the `rtd-footer-container` node inside the primary sidebar.
```